user-console/src/main/java/org/pentaho/mantle/client/MantleApplication.java

```diff
@@ -20,10 +20,7 @@
 
 package org.pentaho.mantle.client;
 
-import com.google.gwt.core.client.GWT;
-import com.google.gwt.core.client.JavaScriptObject;
-import com.google.gwt.core.client.JsArray;
-import com.google.gwt.core.client.JsonUtils;
+import com.google.gwt.core.client.*;
```

Comment:
Dont't use star imports `.*.` your penatho checktyle should have prevented this.